bpf/lib/common.h

```diff
@@ -335,6 +335,14 @@ enum {
 	CT_RELATED,
 };
 
+/* filter_dns returns true if the packet is a DNS response, false otherwise. */
+static inline bool filter_dns(__u8 nexthdr, __u16 dport, __u8 dir)
+{
+	/* The dport represents the destination of the connection. */
+	return ((nexthdr == IPPROTO_UDP || nexthdr == IPPROTO_UDP) &&
```

Comment:
This looks wrong:
`(nexthdr == IPPROTO_UDP || nexthdr == IPPROTO_UDP)` Did you mean to include TCP?

Reply:
Fixed.